template-widget-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "template-react-ts",
-  "version": "1.2.5",
+  "version": "1.3.0",
   "private": true,
   "scripts": {
```

Comment:
Look like there is some missing dependencies when running these jobs

Reply:
The scripts seems to work correctly on my side